### PR TITLE
[16.0][MIG] email_template_qweb: Migration to 16.0

### PR DESCRIPTION
**Pull request by dzungtran89**

### CODE REVIEW

**Comment by dzungtran89** [comment]
@HoKhanhNguyen99 your review is welcome

File: email_template_qweb/readme/CREDITS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
```
Comment (lehoangan) [inline email_template_qweb/readme/CREDITS.rst:1]: Do we need to change it to `The migration of this module from {{from_version}} to {{to_version}} was financially supported by Camptocamp
`

File: email_template_qweb/migrations/15.0.1.0.0/pre-migration.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
```
Comment (lehoangan) [inline email_template_qweb/migrations/15.0.1.0.0/pre-migration.py]: Do we need to remove this folder?

Comment (dzungtran89) [reply email_template_qweb/migrations/15.0.1.0.0/pre-migration.py]: yep, ty, I updated the PR

File: email_template_qweb/readme/CREDITS.rst (hunk 1 above)
Comment (dzungtran89) [reply email_template_qweb/readme/CREDITS.rst:1]: ah, no need, because the file has already been existed

**Review by lehoangan — APPROVED** [review]
LGTM

**Pull request closed without merging**